src/language-js/print/function.js

```diff
@@ -211,6 +211,7 @@ function printArrowFunctionExpression(path, options, print, args) {
       }
       if (
         isCallLikeExpression(parent) ||
+        parent.type === "ImportExpression" ||
```

Comment:
Why not add  to  `isCallLikeExpression`

Reply:
Because AST form is different between `ImportExpression` and `CallExpression` and `isCallLikeExpression` is used for not only printing curried functions. For example, we use `node.arguments` after checking with `isCallLikeExpression` (https://github.com/prettier/prettier/pull/9992/files#diff-112b7d78917580aa0cc6cc3ac0e641c6a9541c0ed4de574f05dba2ae51480ccaR525-R526)